release_build_files/readme.md

```diff
@@ -618,6 +636,10 @@ code.
 ## Release Notes
 ### Upcoming Release
 -   Changes
+    - GMA: Added the Google Mobile Ads SDK with updated support for AdMob. See
+      the [Get Started
+      Guide](https://firebase.google.com/docs/admob/cpp/quick-start) for more
+      information.
```

Comment:
Another line item for AdMob: Deprecated the `firebase::admob`. Please use the new SDK in `firebase::gma` instead. This can be in the deprecation PR.

Reply:
Yes, we agreed upon 

```
   - AdMob: The AdMob SDK has been deprecated. Please update your app to
      use the new Google Mobile Ads SDK which facilitates similar functionality.
```

in the deprecation PR.